dace/libraries/blas/nodes/gemm.py

```diff
@@ -1010,15 +1010,30 @@ def validate(self, sdfg, state):
             if dst_conn == '_a':
                 subset = dc(memlet.subset)
                 subset.squeeze()
-                size0 = subset.size()
+                size0 = []
```

Comment:
How about we add to `dace.symbolic.evaluate` a `throw_error` flag with default value `True.` When set to `False,` it returns the symbolic expression instead of raising `TypeError.` Then you could rewrite this (and everything below) as follows:
```python
size0 = dace.symbolic.evaluate(subset, sdfg.constants, throw_error=False)
```
I will bring it up in the next meeting.

Reply:
Using `dace.symbolic.simplify` with a symbols map sounds like an elegant solution for this. The `throw_error` flag may be a bit confusing imo.